docs/content/topics/operators/security.md

```diff
@@ -19,28 +19,35 @@ For that reason, it is prudent to configure security.
   in a Helm install, do `helm install --namespace brigade ...`.
 - *RBAC Enabled*: When installing with Helm, role-based access control is enabled by default.
   To disable,`--set rbac.enabled=false` will turn off role-based access control.
-- *Do not run more than one brigade per namespace*: Running multiple installs of Brigade
+- *Do not run more than one Brigade per namespace*: Running multiple installs of Brigade
   in the same namespace can cause naming collisions which could result in
   unauthorized access to pods.
-- *Do not run multiple tenants against the same brigade*: Brigade does not implement
-  security controls that allow multiple tenants to share the same Brigade instance.
-  Brigade supports multiple projects per Brigade server instance, but those projects
-  should be owned by the same tenant.
 
 ## API Server Security
 
-TODO: update
+If Brigade's API server will be exposed to the internet, e.g. via an external
+IP (recall that Brigade's default service type in the [Brigade chart] is
+`LoadBalancer` and so a public-facing IP will be provisioned), care should be
+taken to secure inbound communication. Minimally, TLS should be enabled and 
+SSL certificates should not be self-signed. You may also consider using an
+[Ingress Controller] for an extra layer of traffic filtering and/or protection.
```

Comment:
I'm not sure that an ingress controller confers any additional degree of protection.

Perhaps consider rewording what precedes this. Something like:

> If Brigade's API server will be exposed to the internet either via a service of type `LoadBalancer` having a public IP or via an ingress controller, care should...